forge-gui/res/cardsfolder/upcoming/captain_rex_nebula.txt

```diff
@@ -0,0 +1,14 @@
+Name:Captain Rex Nebula
+ManaCost:1 R W
+Types:Legendary Creature Human Pilot Employee
+PT:2/2
+T:Mode$ Phase | Phase$ BeginCombat | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigAnimate | TriggerDescription$ At the beginning of combat on your turn, choose target nonland permanent you control. Until end of turn, it becomes a Vehicle artifact with base power and toughness each equal to its mana value, and it gains crew 2 and "Crash Land — Whenever this Vehicle deals damage, roll a six-sided die. If the result is equal to this Vehicle's mana value, sacrifice this Vehicle, then it deals that much damage to any target."
+SVar:TrigAnimate:DB$ Animate | ValidTgts$ Permanent.nonLand+YouCtrl | TgtPrompt$ Select target nonland permanent you control | RemoveCardTypes$ True | Power$ X | Toughness$ X | Types$ Vehicle,Artifact | Keywords$ Crew:2 | Triggers$ CrashLand
+SVar:CrashLand:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player,Permanent | Execute$ RollCounters | TriggerZones$ Battlefield | TriggerDescription$ Crash Land — Whenever this Vehicle deals damage, roll a six-sided die. If the result is equal to this Vehicle's mana value, sacrifice this Vehicle, then it deals that much damage to any target.
```

Comment:
The wording on such triggers is quite narrow, i.e. you need to use `DamageDealtOnce`
Sometimes they also use rulings as reminders:
> The Crash Land ability will trigger once for each instance of damage, no matter how many players or permanents the Vehicle deals damage to. For example, if the Vehicle is blocked by two creatures and it deals damage to each of them at the same time, the Crash Land ability will trigger only once.